packages/jest-util/src/index.js

```diff
@@ -87,6 +87,19 @@ const warnAboutUnrecognizedOptions  = (argv: Object, options: Object) => {
   }
 };
 
+/**
+ * Normalizes relative paths to always use "/", even on Windows where "\" is the
```

Comment:
I feel like it'd be useful outside of the tests in Jest though (eg. for other projects that have file paths in their unit test snapshots)